packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -544,6 +544,7 @@ export const getDocLinks = ({ kibanaBranch }: GetDocLinkOptions): DocLinks => {
       lowercase: `${ELASTICSEARCH_DOCS}lowercase-processor.html`,
       pipeline: `${ELASTICSEARCH_DOCS}pipeline-processor.html`,
       pipelines: `${ELASTICSEARCH_DOCS}ingest.html`,
+      csvPipelines: `${ELASTIC_WEBSITE_URL}guide/en/ecs/${DOC_LINK_VERSION}/ecs-converting.html`,
```

Comment:
@alisonelizabeth @lcawl For some reason, this link doesn't work when I test this locally. Any tips to help me get unstuck? Thanks in advance. 🙇

Reply:
I tested just now, and the "Create pipeline from CSV Docs" link works if I update the documentation.ts file to use a pre-existing URL, for example:
> this.createPipelineCSVUrl = links.ingest.pipelines;

... So this seems to imply the problem is on the doc link service side.

Reply:
> For some reason, this link doesn't work when I test this locally

Did you rebuild the packages? Now that the links are in a module, you need to `yarn run kbn bootstrap` for the changes to be taken into account